test/extended/authorization/scc.go

```diff
@@ -21,7 +21,7 @@ var _ = g.Describe("[sig-auth][Feature:SCC][Early]", func() {
 
 	oc := exutil.NewCLIWithoutNamespace("working-scc-during-install")
 
-	g.It("should not have pod creation failures during install", func() {
+	g.It("should not have pod creation failures during install [apigroup:config.openshift.io]", func() {
```

Comment:
What requires this api group in this test?

Reply:
`util.AllClusterVersionsAreGTE` and https://github.com/ingvagabund/origin/blob/7826178007475886fb8878e1e60e91f14eb476b2/test/extended/util/framework.go#L1986:
```
func AllClusterVersionsAreGTE(version semver.Version, config *rest.Config) (bool, error) {
	c, err := configv1client.NewForConfig(config)
	if err != nil {
		return false, err
	}
	cv, err := c.ConfigV1().ClusterVersions().Get(context.TODO(), "version", metav1.GetOptions{})
	if err != nil {
		return false, err
	}
```

Reply:
Fun.  Lucky for us, we can remove that line now.

Reply:
Removed